grafonnet-base/helpers.libsonnet

```diff
@@ -4,15 +4,20 @@ local xtd = import 'github.com/jsonnet-libs/xtd/main.libsonnet';
 {
   local root = self,
 
+  // Grabs last item in an array
   last(arr): std.reverse(arr)[0],
+
+  // Returns the whole array except the last item
   allButLast(arr): arr[0:std.length(arr) - 1],
 
+  // Gets the content from source on a specified JSONPath
   getContent(source, path):
     local content = xtd.jsonpath.getJSONPath(source, path);
     if content == null
     then self.last(xtd.string.splitEscape(path, '.'))
```

Comment:
Why return last element of path if no content found?

Reply:
I'm covering for missing docstrings here, I agree, this needs to be implemented better but I don't know how yet.

Reply:
And found how to deal with it: https://github.com/grafana/grafonnet/pull/56